commands/🔰 Info/support.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
         .setTitle(":heart: Join here!")
         .setFooter(ee.footertext, ee.footericon)
         .setURL(`https://discord.gg/kUuNQwnvCF`)
-        .setDescription(`[Click here](https://discord.gg/kUuNQwnvCF)`)
+        .setDescription(`[Click here](https://discord.gg/kUuNQwnvCF) (https://discord.gg/kUuNQwnvCF)`)
```

Comment:
The link address is shown when you click on it in Discord, no need to add it afterwards